owncloudApp/src/main/java/com/owncloud/android/presentation/files/filelist/MainFileListViewModel.kt

```diff
@@ -94,6 +94,7 @@ class MainFileListViewModel(
     fileListOptionParam: FileListOption,
 ) : ViewModel() {
 
+    var lastVisibleItemPositions: IntArray? = null
```

Comment:
It's usually less secure to expose variables out of the ViewModel as in this case... but let's see how things go after resolving the previous comments 😉